docs/resources/projects/how-to-create-find-organize-projects.md

```diff
@@ -4,6 +4,7 @@ title: How to Create, Find, and Organize Projects
 description: Learn how to create, find, and organize projects in FlutterFlow to streamline your app development process.
 tags: [Project Management, FlutterFlow, Organization, Projects]
 toc_max_heading_level: 5
+sidebar_position: 3
```

Comment:
Create project should be just after what is a project. So this should be second in order.

Reply:
Updated!